postgres-matrix.yml

```diff
@@ -3,4 +3,4 @@ project_name: hll
 
 version_matrix:
   - 2.16:
```

Comment:
Version 2.16 does not support PG15. 2.17 does.

Reply:
Also we need PG10 here as we wish to create packages for that PG version as well. Do we have the necessary docker images for PG10?

Reply:
Fixed wrong mapping and added pg 10